geomesa-tools/pom.xml

```diff
@@ -66,6 +66,30 @@
             <artifactId>commons-csv</artifactId>
         </dependency>
 
+        <!-- Kafka CLI dependencies -->
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-kafka-datastore</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_2.11</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j:*</groupId>
+                    <artifactId>slf4j-log4j12</artifactId>
```

Comment:
@Afterglow375 Actually just make the artifact id a ```*``` and leave the group id as ```org.slf4j``` the other syntax of ```org.slf4j:*``` is a shorthand syntax for ```groupId:artifactId:version``` i believe